src/Microsoft.AspNetCore.Server.Kestrel.Core/KestrelServerLimits.cs

```diff
@@ -207,5 +209,14 @@ public TimeSpan RequestHeadersTimeout
                 _maxConcurrentUpgradedConnections = value;
             }
         }
+
+        public TimeSpan RequestBodyReadTimeout
+        {
+            get => _requestBodyReadTimeout;
```

Comment:
But if we go with this, it doesn't look like we need a backing field. Can be simplified to 
```c#
public TimeSpan RequestBodyReadTimeout { get; set; } = TimeSpan.FromSeconds(60);
```